src/sources/redis/channel.rs

```diff
@@ -28,6 +29,7 @@ pub async fn subscribe(
     redis_key: Option<String>,
```

Comment:
None of the args here jump out as something that can be pulled out and reduced for the function, I'd be ok with a clippy ignore here.